tests/unit/api/test_token.py

```diff
@@ -145,7 +145,7 @@ class User:
                 id=1213,
                 username="read_user",
                 password="test",
-                scopes=[Scope("read:targets")],
+                scopes=[],
```

Comment:
nit: It mocks the database with a "read_user" -- even our authentication/authorization built-in feature doesn't support multiple users (only "admin")
Do we want to test it with a user that doesn't have any scope?
On line 164 we request a token for a user with `write:targets`. I would suggest giving the user at least one valid scope (`read:settings`) for example.